### DISCUSSION TITLE
Adding RSTPs from other networks

### DISCUSSION THREAD

**Discussion by CancunManny** [blakeblackshear/frigate#14673] [Ask A Question]
I understand this is not really a Frigate question, but perhaps some of you with much more knowledge than me can tell me if it is possible, and if so give me some keywords I can use to get the full instructions.

If I wanted to use Frigate in a small private street, about 30 homes, 15 on each side of the street.  Would it be possible without much effort to get RTSP from different homes each with their own network?  Is it possible to just VPN the RTSP or something similar?  The goal would be for each network to continue to be independent of one another but somehow get their RTSP feed onto one single Frigate.

Or would the better solution be to extend wifi coverage to cover everyone's front yard so all the cameras log in to the same network that way?

**Comment by SeanChengN** [comment]
After testing, you can use VPNs such as ZeroTier.

**Reply by CancunManny** [reply]
thanks will read up on ZeroTier to see if it is a viable option for my case.

**Comment by aaronjrodrigues** [comment]
VPNs like Tailscale would be perfect for this since they also do nat traversal which means that it’ll work even when you’re behind a cgnat

**Comment by abufreeh** [comment]
did not work for me ! ... do  I need to do more than install a client and join to network?
I have debian with docker .. and frigate

**Comment by aaronjrodrigues** [comment]
You would need to expose the subject where the devices are located, search up Tailscale subnet routing.

-aaron
